susemanager-utils/susemanager-sls/src/modules/kiwi_info.py

```diff
@@ -127,6 +127,13 @@ def parse_kiwi_result(dest):
         )
         if result["retcode"] == 0:
             ret = json.loads(result["stdout"])
+        else:
+            # pylint: disable-next=undefined-variable
+            result = __salt__["cmd.exec_code_all"](
+                "/usr/bin/python3.11", _kiwi_result_script, args=[path]
```

Comment:
Now that python3.11 is almost everywhere, shouldn't we start with it and only try python3 on failure?